src/pages/TagCreator/TopicTagSection.js

```diff
@@ -143,28 +149,58 @@ export const NewTags = ({ resetForm, setDisplayState, tagsToAdd, setChangeValue,
-      </Grid>
-      <Grid item xs={12} sm={10} style={{ paddingBottom: '20px' }}>
-        <Typography variant='h6' style={{ fontWeight: '400' }} className={classes.typoStyle}>If you don’t see the <SettingsGearIcon />  button it means you don’t have “edit repository settings” privileges (and can’t perform the steps below).
-        Please click <Link component="button" variant='h6' onClick={() => handleQueryParamLink()} underline='always' style={linkStyles}>{clipboard.copied ? 'Copied' : 'here'}</Link> to copy this page link and send it to your repository admin.</Typography>
+      <Grid container>
+        <Grid style={{ padding: '24px 0px' }}>
+          <Typography variant='h4'>How to add your tags to your project’s repository</Typography>
+        </Grid>
+        <Grid item xs={12} sm={10}>
+          <Typography variant='body1' className={classes.typoStyle}>
+                1. Under your <Link target="_blank" href={repositoryUrl} underline='always' className={classes.lStyle}  >project’s repository,</Link> click <SettingsGearIcon /> to paste your tags.
+          </Typography>
+        </Grid>
+        <Grid item xs={12} sm={10} style={{ padding:'24px 16px 0' }}>
+          <Typography variant='body1' className={classes.typoStyle}>If you don’t see the <SettingsGearIcon />  button it means you don’t have “edit repository settings” privileges (and can’t perform the steps below).
+              Please click <Link component="button" variant='body1' onClick={() => handleQueryParamLink()} underline='always' className={classes.lStyle} >{clipboard.copied ? 'Copied' : 'here'}</Link> to copy this page link and send it to your repository admin.</Typography>
+        </Grid>
+        <ImageComponent src='/images/instructions-step1.svg' alt='Step 1' />
+        <Grid>
+          <Typography variant='body1' className={classes.typoStyle} >2. Under <Box component="span" style={{ fontWeight: '700' }}>&quot;Topics&quot;</Box>, paste the topic you want to add to your repository. </Typography>
+        </Grid>
+        <Grid item xs={12} sm={10} style={{ padding: '20px' }}>
+          <Typography variant='body1' className={classes.typoStyle}>Click each individual generated topic tag to copy it one at a time. Paste selected tag into your repository.</Typography>
+        </Grid>
+        <Grid container style={{ paddingLeft: '20px' }}>
+          <Grid item>
+            <Grid container direction="row" alignItems="center" >
+              <CopyPasteTopicTag topicnames={copyPasteTopicTags} variant='outlined' />
+            </Grid>
+          </Grid>
+        </Grid>
+        <ImageComponent src='/images/instructions-step2.svg' alt='Step 2' />
+        <Grid>
+          <Typography variant='body1' className={classes.typoStyle} >3. After you have finished adding your tags, click <Box component="span" style={{ fontWeight: '700' }}>Save Changes</Box> </Typography>
+        </Grid>
+        <ImageComponent src='/images/instructions-step3.svg' alt='Step 3' />
```

Comment:
Would it be possible to format this in prettier, or would it cause a conflict with eslint?

Reply:
@bruceplai , It is different from How to use. This has copy-paste tags and also some more instructions.